be/src/storage/tablet_updates.cpp

```diff
@@ -251,6 +251,13 @@ Status TabletUpdates::_load_from_pb(const TabletUpdatesPB& tablet_updates_pb) {
     std::unique_lock l1(_lock);
     std::unique_lock l2(_rowsets_lock);
 
+    std::vector<TabletSegmentId> tsids;
```

Comment:
What's the error raised?
It's like this one?
```
java.sql.SQLException: (conn=448) Bad page: invalid footer, file=/opt/starrocks/be/storage/data/812/2360503/1912606354/020000000003b5eeff491c6513959632d141c7e3baca8c9d_0.dat
```

Reply:
> What's the error raised? It's like this one?
> 
> ```
> java.sql.SQLException: (conn=448) Bad page: invalid footer, file=/opt/starrocks/be/storage/data/812/2360503/1912606354/020000000003b5eeff491c6513959632d141c7e3baca8c9d_0.dat
> ```

No, the error maybe like `actual row size changed` or return error query result. It does not cause disk file error but maybe lost some data.